main.go

```diff
@@ -116,6 +121,14 @@ func main() {
 		"The address the health endpoint binds to.")
 	flag.IntVar(&webhookPort, "webhook-port", 9443,
 		"Webhook Server port (set to 0 to disable)")
+	flag.IntVar(&baremetalhostConcurrency, "baremetalhost-concurrency", 10,
+		"Number of baremetalhosts to process simultaneously")
+	flag.IntVar(&preprovisioningimageConcurrency, "preprovisioningimage-concurrency", 10,
+		"Number of preprovisioningimages to process simultaneously")
+	flag.IntVar(&hostfirmwaresettingsConcurrency, "hostfirmwaresettings-concurrency", 10,
+		"Number of hostfirmwaresettings to process simultaneously")
+	flag.IntVar(&bmceventsubscriptionConcurrency, "bmceventsubscription-concurrency", 10,
+		"Number of bmceventsubscriptions to process simultaneously")
```

Comment:
Why would anybody want to control these independently?

Reply:
Just adding a flag, not imposing that anybody should be controlling this independently.

Reply:
We _are_ imposing that they have to be controlled independently because there is no option to allow them all to be set together (other than the existing env var).
If there's no reason to control them independently (and I can't think of one) then we're just making it needlessly hard for the user (and ourselves) and moving even further away from any kind of standard way to do this in kubebuilder-generated controllers.

Reply:
I will remove this since its not what we want and keep one flag for BMH controller only but IMO, controlling the concurrency of different controllers with the flag value of one controller, isn't standard way either.